src/test/results_test.go

```diff
@@ -169,11 +169,6 @@ func TestGoIgnoreUnknownOutput(t *testing.T) {
 	assert.Equal(t, 0, results.Skips())
 }
 
-func TestGoFailIfUnknownTestPasses(t *testing.T) {
```

Comment:
I think go-junit-report handles the case we test here (of results happening out of order). I'm not sure to what degree we need to care.